lib/GADS/Column/String.pm

```diff
@@ -26,6 +26,11 @@ extends 'GADS::Column';
 
 with 'GADS::Role::Presentation::Column::String';
 
+sub purge {
```

Comment:
I'm not entirely sure what this is for - it may be an old debug value that was placed in before we moved to using roles - I shall take a look.